.github/workflows/build-test-release.yml

```diff
@@ -175,21 +210,36 @@ jobs:
         run: |
           sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
           curl -sSL https://install.python-poetry.org | python3 -
-          poetry install -E docker
-          poetry run pytest -v --splunk-version=${{ matrix.splunk-version }} -m doc
+          poetry install
+          poetry run pytest -v --splunk-version=${{ matrix.splunk-version }} -m doc tests/psa_tests
 
   test-splunk-matrix:
     name: Test Matrix
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        splunk-version: [8.1, 8.2]
+        test-marker: ['test_report', 'splunk_connection_kubernetes', 'splunk_app_fiction', 'splunk_app_broken','splunk_app_cim_fiction','splunk_app_cim_broken','splunk_fiction_indextime','splunk_fiction_indextime_broken','splunk_setup_fixture','splunk_app_requirements','splunk_app_requirements_modinput','splunk_app_requirements_uf','splunk_app_requirements_scripted']
+        splunk-version: ['8.1','8.2']
+        os: [ubuntu-latest]
```

Comment:
I think it was intended to run on all OS-es, but the configuration was incorrect, are we only supporting Ubuntu now?

Reply:
Actually, found an original code for this: https://github.com/splunk/pytest-splunk-addon/pull/426/files#diff-227729f71180f01a43dec20798948144523f45c9e4f23555469002e7178ce4b4R197-R205

Reply:
Previously it was only executing on ubuntu-latest even though mac-os and windows has been passed. A JIRA has been created to implement the functionality to test on windows and mac in CI/CD.
Ref: https://splunk.atlassian.net/browse/AQA-69